### ISSUE TITLE
New logo hardly visible with current theme

### ISSUE THREAD

**Issue by tintou** [mesonbuild/meson#4930]
On http://mesonbuild.com/ the logo doesn't meet the accessibility standards. You might want to consider using the [hotdoc_lumen_theme](https://github.com/hotdoc/hotdoc_lumen_theme) or have a [custom bootstrap theme](https://github.com/hotdoc/hotdoc_bootstrap_theme) that could be using Meson colors for links.

**Comment by mikkelfj** [comment]
https://groups.google.com/forum/#!topic/mesonbuild/5vuZWs_nDsk

**Comment by tintou** [comment]
@mikkelfj Is it a paper chase :smiley: because there is no backlog of what could've happened IRL

**Comment by nirbheek** [comment]
We could also make the upper left corner be a rectangular white background. That way we can have dark mode usability for the rest of the paeg without destroying the icon's contrast.

**Comment by Akaricchi** [comment]
I'd personally prefer a brighter theme as well. Was never a fan of the switch to the dark one in the first place.

**Comment by bonzini** [comment]
Fixed by commit d31db565e1405b303390db6502da959e4f5fe62f
